.github/scripts/deploy_nightly_pypi.py

```
@@ -0,0 +1,258 @@
+
+
+def get_current_temoa_version() -> str:
+    """Reads the base version from pyproject.toml."""
+    with open('pyproject.toml') as f:
+        data = tomlkit.parse(f.read())
+    return data['project']['version']
+
+
+def generate_nightly_version(base_version: str) -> str:
+    """Generates a PEP 440 compliant nightly version string."""
+    today = datetime.now(timezone.utc).strftime('%Y%m%d')
```

Ruff (0.14.5)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)